src/bootstrap/setup.rs

```diff
@@ -176,6 +176,14 @@ pub fn setup(config: &Config, profile: Profile) {
         );
     }
 
+    if profile == Profile::Tools {
+        eprintln!();
+        eprintln!(
+            "note: the `tools` profile uses the `stage2` toolchain (use \
```

Comment:
is *uses* right here? I would have mildly expected "produces" in some sense (even if that production is via download from previous CI builds)

Reply:
"sets up" could be used instead -- produces sounds a bit weird (not to mention a bit formal considering this is targeted at newer users)?

I thought about "builds" or "downloads" but I feel like that will lead to more confusion as, as I currently understand it, both occur (stage2 compiler + std are downloaded, tools are built).

Reply:
Sure, any of those seem reasonable. r=me with "sets up" and commits squashed.